python/mxnet/gluon/contrib/estimator/event_handler.py

```diff
@@ -305,15 +293,22 @@ def batch_end(self, estimator, *args, **kwargs):
 
     def epoch_begin(self, estimator, *args, **kwargs):
         if self.verbose >= self.LOG_PER_EPOCH:
+            is_training = False
+            for metric in self.metrics:
+                if 'training' in metric.name:
```

Comment:
Please add a comment and describe that this test always works thanks to the hack in `Estimator.__init__`

Reply:
I'll comment this.